app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -240,6 +234,19 @@ class HomeFragment : Fragment() {
         return view
     }
 
+    /**
+     * The [SessionControlView] is forced to update with our current state when we call [HomeFragment.onCreateView]
+     * in order to be able to draw everything at once with the current data in our store. The [View.consumeFrom] coroutine dispatch
+     * doesn't get run right away which means that we won't draw on the first layout pass.
+     * */
+    fun updateSessionControlView(view: View) {
+        sessionControlView?.update(homeFragmentStore.state)
```

Comment:
nit: I like the inner function version I suggested because it makes it clear that the code is repeated and prevents someone from missing that if they decide to refactor this. But I can go either way